Casks/m/mozilla-vpn.rb

```diff
@@ -0,0 +1,32 @@
+cask "mozilla-vpn" do
+  version "2.28.0"
+  sha256 "26e8daba2845d1aa655feb206f0db7e3fb24162f9ea8a1d5344fc3c0c6182a31"
+
+  url "https://archive.mozilla.org/pub/vpn/releases/#{version}/mac/MozillaVPN.pkg"
+  name "Mozilla VPN"
+  desc "Fast, secure and easy to use VPN. Built by the makers of Firefox"
```

Comment:
```suggestion
  desc "VPN client"
```

We try to keep the description as concise as possible (see [documentation](https://docs.brew.sh/Cask-Cookbook#stanza-desc)).